reactive/kotlinx-coroutines-reactor/test/FluxContextTest.kt

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2016-2019 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
```

Comment:
Copyright year.
By the way, you can use `$today.year` in copyright template

Reply:
Good catch. Thanks.